sdks/python/container/Dockerfile

```diff
@@ -43,9 +40,12 @@ RUN  \
        # Required for using Beam Python SDK on ARM machines.
        libgeos-dev \
        && \
-    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /var/lib/apt/lists/*
 
-    pip install --upgrade pip setuptools wheel && \
+RUN python -m venv venv-beam
```

Comment:
Beam SDK workers create a venv as well, which is superimposed upon the default environment. I think having Beam dependencies packages in a separate venv would break this setup.

Reply:
This is used to hold all the Python packages we need and later it is copied to the distroless image. I will check whether it works with DataflowRunner later.

Reply:
I removed this to simply the process.